ci-operator/step-registry/ipi/conf/capability/baselinecaps/v-capset/ipi-conf-capability-baselinecaps-v-capset-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+openshift-install version
+declare -a baseline_caps_set_array
+IFS="," read -r -a baseline_caps_set_array <<< "$(openshift-install explain installconfig.capabilities.baselineCapabilitySet | grep 'Valid Values' | awk -F':' '{print $2}' | tr -d '"')"
+
+if [[ ${#baseline_caps_set_array[@]} -eq 0 ]]; then
+    echo "Unable to get valid values for baselineCapabilitySet!"
+    exit 1
+fi
+
+#Remove None from array
+baseline_caps_set_array=( "${baseline_caps_set_array[@]/None/}" )
```

Comment:
If we extend the test strategy, sounds we even can include "None", so we can cover them togeher.
1. bsaelineCaps: v4.x/vCurrent/None
2. baselineCaps: v4.x/vCurrent/None + additionalEnabledCapabilites